cristin-import/src/test/java/no/unit/nva/cristin/CristinDataGenerator.java

```diff
@@ -191,10 +196,25 @@ private CristinObject createObjectWithCristinContributorRoleCode(Integer index,
                 .withEntryCreationDate(LocalDate.now())
                 .withPublicationYear(randomYear())
                 .withContributors(contributors)
+                .withBookReport(randomBookReport())
                 .withPublicationOwner(HardcodedValues.HARDCODED_PUBLICATIONS_OWNER)
                 .build();
     }
 
+    private List<CristinBookReport> randomBookReport() {
+        CristinBookReport bookReport = new CristinBookReport().copy().build();
+        bookReport.setIsbn(randomIsbn13());
+        bookReport.setPublisherName(randomString());
+        bookReport.setNumberOfPages(randomString());
+        List bookReportList = new ArrayList();
```

Comment:
Although you can use List.of(...) here, in general try not use generic classes (e.g List<...> ) without specifying the type. Try to always specify the type because when you will be comparing two lists they will not be equal if one is List<Object> and the other List<SpecificType> even if they contain the same objects.